R/dataset-list.R

```diff
@@ -0,0 +1,40 @@
+source(here::here("R", "entities.R"))
+
+datasets <- c(
+  Region$new(name = "afghanistan",
+             case_modifier = function(cases) {
+               cases <- cases[!is.na(iso_3166_2)]
+               return(cases)
+             },
+             stable = FALSE),
+  Region$new(name = "belgium"),
+  Region$new(name = "brazil"),
+  Region$new(name = "canada"),
+  Region$new(name = "colombia"),
+  Region$new(name = "germany"),
+  Region$new(name = "india"),
+  Region$new(name = "italy"),
+  Region$new(name = "russia"),
+  Region$new(name = "united-kingdom",
+             covid_regional_data_identifier = "UK"),
+  Region$new(name = "united-states",
+             covid_regional_data_identifier = "USA",
+             region_scale = "State"),
+  SuperRegion$new(name = "cases",
+                  case_modifier = function(cases) {
+                    cases <- cases[, .(region = country, date = as.Date(date), confirm = cases_new)]
+                    cases <- cases[date <= Sys.Date()]
+                    cases <- cases[, .SD[date <= (max(date) - lubridate::days(3))], by = region]
+                    cases <- cases[, .SD[date >= (max(date) - lubridate::weeks(12))], by = region]
+                    data.table::setorder(cases, date)
+                  }),
+  SuperRegion$new(name = "deaths",
```

Comment:
good point well made, it won't

Reply:
I think it should be a fairly simple task of adjusting the data processing function used here to first sum and bind a global sum. If you have a chance to look at it then that would be great!

Reply:
I have tried to pull this in to reflect the changes. It "sadly" required the addition of more control over the folder naming so that you can manipulate national/regional and also put two different sets into .../deaths/...

Reply:
So I was thinking we would want the global/region estimates in a different top level folder (currently in `region/cases` and `region/deaths`)?

Reply:
This branch will produce:

### from superregion dataset definitions
national/cases/[national|summary]
national/deaths/[national|summary]
region/deaths/[region|summary] (I only just spotted the tail folder for this - it was region/deaths/[national|summary])
### from region dataset definitions
subnational/<country>/cases/[national/summary]

Reply:
Excellent - looks great.